src/interfaces.ts

```diff
@@ -407,7 +407,7 @@ export interface Web3AuthOptions {
   /**
    * Set this flag to false to generate keys on client side
    * by default keys are generated on using dkg protocol on a distributed network
-   * @defaultValue undefined
+   * @defaultValue true if keyType is ed25519, false for secp256k1 keys
```

Comment:
```suggestion
   * @defaultValue false if keyType is ed25519, true for secp256k1 keys
```

should be enabled by default for secp256k1. i think it is a bit problematic that we have a different default here depending on the key type.

couldn't we just default to true in both cases but clearly state in the documentation that this should be set to false if you need key exportability?

Reply:
we are doing pretty much same in tkey and torus.js as well. We have to change  `secp256k1` their first. Plan is to make it default in future , we are starting with ed25519 as if something goes wrong impact will be least.